src/apps/material/helper.ts

```diff
@@ -512,23 +512,32 @@ export const getBlacklistedQueryArgs = (
   const args = {
     recordid: faustIds
   };
-  // Return query args with the either availability or pickup branches excluded.
-  if (blacklistType !== "both") {
+
+  if (blacklistType === "availability") {
     return {
       ...args,
-      ...formatBranches([branchesFromConfig(blacklistType, config)])
+      ...formatBranches([branchesFromConfig("availability", config)])
     };
   }
 
-  // If we want to blacklist both availability and pickup branches
-  // return query args with both blacklist types excluded.
-  return {
-    ...args,
-    ...formatBranches([
-      branchesFromConfig("availability", config),
-      branchesFromConfig("pickup", config)
-    ])
-  };
+  if (blacklistType === "pickup") {
+    return {
+      ...args,
+      ...formatBranches([branchesFromConfig("pickup", config)])
+    };
+  }
+
+  if (blacklistType === "both") {
+    return {
+      ...args,
+      ...formatBranches([
+        branchesFromConfig("availability", config),
+        branchesFromConfig("pickup", config)
+      ])
+    };
+  }
+
+  return args;
```

Comment:
Please reconsider this change.

Reading through the code I cannot see how the updated implementation if different from the current one - except being more verbose.